src/HotChocolate/Fusion-vnext/src/Fusion.Composition/Properties/CompositionResources.resx

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <root>
-  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+              xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
```

Comment:
Are these changes caused by automated formatting?

Reply:
I think so yes. Is there a DotSettings I'm missing for resharper?

Reply:
I don't have this format on save behaviour (Rider on Windows).

Can you please revert the unrelated changes?